YIF.Core.Service/Concrete/Services/IoEAdminService.cs

```diff
@@ -342,5 +342,17 @@ public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteIoEModera
 
             return result.Set(new DescriptionResponseApiModel(_resourceManager.GetString("LectorWasAdded")), true);
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteIoELector(string lectorId)
+        {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            var lector = await _lectorRepository.Get(lectorId);
+            if (lector == null)
+            {
+                throw new NotFoundException($"{_resourceManager.GetString("IoELectorWithSuchIdNotFound")}: {lectorId}");
+            }
+            await _lectorRepository.Delete(lector.Id);
```

Comment:
you should drop user's role "Lector" to disable user's activity under this role after deleting

Reply:
resource IoELectorWithSuchIdNotFound is correct? i guss not pls create correct line something like lector or user with such Id isn't exist

Reply:
Fixed